src/Helper/Helper_PDF_List_Table.php

```diff
@@ -332,7 +336,7 @@ public function column_name( $item ) {
 
 		?>
 
-		<a href="<?= $edit_url; ?>"><strong><?= rgar( $item, 'name' ); ?></strong></a>
+		<a href="<?= $edit_url; ?>" aria-label="<?= $pdf_name ?> PDF"><strong><?= $pdf_name ?></strong></a>
```

Comment:
Make translatable. Also need to run attributes through `esc_attr()` for security https://developer.wordpress.org/reference/functions/esc_attr/

Reply:
Updated.